src/tools/pkcs11-tool.c

```diff
@@ -6035,7 +6035,6 @@ static EVP_PKEY *get_public_key(CK_SESSION_HANDLE session, CK_OBJECT_HANDLE priv
 			OSSL_PARAM_free(params);
 #endif
 			return pkey;
-		case CKK_DSA:
```

Comment:
Maybe we could keep this one if somebody would need to use the pkcs11-tool with some other module that supports DSA.